Major Editorial Update (#826)
ethereum-oasis-op/baseline: docs/community/governance.md

OASIS Open is a non-profit, vendor-neutral standards developing organization. It provides rules and guidelines to ensure equitable and transparent oversight of technical collaborations in open source and open standards. Additionally, it provides services such as technical program support, legal, marketing, and events management.

> The Baseline Protocol is governed by the [EEA Community Projects Project Governing Board (PGB)](https://github.com/ethereum-oasis-op/managed-open-project/blob/main/PROJECT-GOVERNING-BOARD.md) under the [OASIS Open Projects Program](http://oasis-open-projects.org/). EEA Community Projects exists to provide a neutral forum for companies and community representatives to create high-quality specifications that facilitate Ethereum’s longevity, interoperability, and ease of integration. One does not have to "join" the group formally or financially in order to make a contribution to Baseline or other Ethereum OASIS projects.

Reviewer: Not EEA any more

```suggestion
The Baseline Protocol is governed by the [Ethereum OASIS Projects Project Governing Board (PGB)](https://github.com/ethereum-oasis-op/managed-open-project/blob/main/PROJECT-GOVERNING-BOARD.md) under the [OASIS Open Projects Program](http://oasis-open-projects.org/). EEA Community Projects exists to provide a neutral forum for companies and community representatives to create high-quality specifications that facilitate Ethereum’s longevity, interoperability, and ease of integration. One does not have to "join" the group formally or financially in order to make a contribution to Baseline or other Ethereum OASIS projects.
```

Additional documentation about the OASIS Open Projects program can be found [here](https://github.com/oasis-open-projects/documentation).

## License and Patent Policies <a id="license-and-patent-policies"></a>

> All repos in the EEA Community Projects organization, including Baseline Protocol repositories, adhere to OASIS Open Projects [license](https://github.com/oasis-open-projects/documentation/blob/master/policy/licenses.md) and [patent polices](https://github.com/oasis-open-projects/documentation/blob/master/policy/call-for-patent-disclosure.md).

Reviewer: Not EEA any more

```suggestion
All repos in the Ethereum OASIS Projects organization, including Baseline Protocol repositories, adhere to OASIS Open Projects [license](https://github.com/oasis-open-projects/documentation/blob/master/policy/licenses.md) and [patent polices](https://github.com/oasis-open-projects/documentation/blob/master/policy/call-for-patent-disclosure.md).
```

> ### Details on OASIS and EEA Community Projects <a id="details-on-oasis-and-eea-community-projects"></a>

Reviewer: ```suggestion
### Details on OASIS and Ethereum OASIS Projects <a id="details-on-oasis-and-eea-community-projects"></a>
```

> Governance documents for [EEA Community Projects](https://github.com/ethereum-oasis-op/managed-open-project/blob/main/GOVERNANCE.md).

Reviewer: rename

```suggestion
Governance documents for [Ethereum OASIS Projects](https://github.com/ethereum-oasis-op/managed-open-project/blob/main/GOVERNANCE.md).
```
